internal/reminder/reminder.go

```diff
@@ -241,3 +260,25 @@ func (r *reminder) adjustCursorForEndOfList(ctx context.Context) {
 		r.repositoryCursor = uuid.Nil
 	}
 }
+
+func createReminderMessages(ctx context.Context, repos []db.Repository) ([]*message.Message, error) {
+	logger := zerolog.Ctx(ctx)
+
+	messages := make([]*message.Message, 0, len(repos))
+	for _, repo := range repos {
+		repoReconcileMessage, err := remindermessages.NewRepoReminderMessage(
+			repo.ProviderID, repo.RepoID, repo.ProjectID,
+		)
+		if err != nil {
+			return nil, fmt.Errorf("error creating reminder message: %w", err)
+		}
```

Comment:
We now stop if any JSON marshalling fails. I expect this to be deterministic, so if one is failing, others will fail too.